hack/test-e2e.sh

```diff
@@ -8,14 +8,13 @@ for test in $( find "${current_dir}/testing" -type f -name 'test-*.sh' | sort);
 	os::log::info "======================================"
 	os::log::info "running e2e $test "
 	os::log::info "======================================"
-	if [ "$(basename $test)" == "test-020-olm-upgrade.sh" ] ; then
-		os::log::warning "Intentionally skipping $test"
-		continue
-	fi
 	if "${test}" ; then
 		os::log::info "e2e $test succeeded at $( date )"
 	else
-		os::log::warning "e2e $test failed at $( date )"
+
+		os::log::error "FAILED FAILED FAILED FAILED FAILED FAILED FAILED"
```

Comment:
is this just for debugging or is this being kept like this?

Reply:
I was intending to leave permanently to make it make it more obvious where it failed